libtlv/src/TlvObjectTests.cpp

```diff
@@ -0,0 +1,100 @@
+//#include "TlvObject.hpp"
+#define BOOST_TEST_DYN_LINK
+#define BOOST_TEST_MAIN
+#define BOOST_TEST_MODULE MyTest
+#include <boost/test/unit_test.hpp>
+#include <vector>
+#include <iostream>
+#include "TlvObject.h"
+
+BOOST_AUTO_TEST_CASE(normal_find)
+{
+	Tlv o;
+	unsigned char t[] = {3, 3, 3};
+	unsigned char tt[] = {0, 0, 11, 0, 2, 4, 4, 0, 0, 13, 0, 1, 2};
+	o.add(12, 0, 3, t);
+	o.add(14, 1, 13, tt);
```

Comment:
What this one means here? I see you've defined library constants, please use them for clarity.